### DISCUSSION TITLE
Problem API and Installation

### DISCUSSION THREAD

**Discussion by Alf20201976** [coinfoundry/miningcore#1243] [Q&A]
Hallo Oliver, I ve an problem , can i do an pool with Miningcore for the VTC?
How can i an installation and configuration on Server VPS Ubuntu ?

Thank you 
Alfonso ... Ich bin in Konstanz.

**Comment by ei8ht187** [comment]
Hello @Alf20201976 
usually a good start is to check the [coins.json](https://github.com/oliverw/miningcore/blob/master/src/Miningcore/coins.json). When your desired coin is listed there then Miningcore supports its. 
To set up a pool follow the instructions given in the readme in the "Building on XYZ" and "Running Miningcore" sections. Last thing you need is a configuration file. Some examples are listed here: https://github.com/oliverw/miningcore/tree/master/examples 

Kind regards

ei8ht

**Reply by Alf20201976** [reply]
Hallo Tank you for u support, I ve done the installation 👍 
[](http://vertcoin-7dminers.com/) , but I ve an problem the mining function, bat i not see on front end the coin,
this pool for the VTC only VTC, can u help me, 


Kind regards
Alfonso

**Reply by ei8ht187** [reply]
Please excuse, but I don't understand what exactly your problem is. So, you've set up the pool but you don't see it in the webUI? If so, the problem is most likley the used WebUI itself (no support for VTC).

And now, as you say you are in Constance, in good ol' german :) 
Entschuldigen Sie bitte, ich verstehe nicht genau was ihr problem ist. Der Pool selbst läuft, nur im Webinterface ist er nicht ersichtlich? Falls ja, liegt das wohl eher am verwendeten WebUI, welches mit VTC nichts anzufangen weiss.

**Reply by Alf20201976** [reply]
Lieber  ei8t187,

Ja, ich bin in Konstanz, das ist mein Schwimmbad http://vertcoin-7dminers.com/, WebUI Ich habe dieses https://github.com/minernl/Miningcore.WebUI, benutze.

Ich habe auch diese https://miningcore.com/ gesehen, wo VTC vorhanden ist.

Sie haben einen Vorschlag für eine WebUI, die für VTC funktioniert.

LG und Vielen Dank  einen schönen Tag.

**Reply by ei8ht187** [reply]
Okey, the problem seems to be something different. The WebUI is unable to find your API. Please check the miningcore configuration if the api is enabled and the configuration of the minersNL WebUI is correct. 

The api part in the config.json for miningcore itself shoud look like this: 
```
    "api": {
        "enabled": true,
        "listenAddress": "*",
        "port": 4000,
        "metricsIpWhitelist": [],
        "rateLimiting": {
            "disabled": true,
            "rules": [
                {
                    "Endpoint": "*",
                    "Period": "1s",
                    "Limit": 5
                }
            ],
            "ipWhitelist": [
                ""
            ]
        }
    },
```
For the WebUI it's best to ask there.

**Comment by Alf20201976** [comment]
Hallo, now function, but can i  not see the payment, I ve 2 block on wallet master,
http://vertcoin-7dminers.com/
